### PR TITLE
[Project] Legacy Build Location Support

### PR DESCRIPTION
**Pull request by alloy**
Replaces #2880.

@samdmarshall What do you think?

### CODE REVIEW

File: lib/cocoapods/project.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ module Pod
   # through the installation process.
   #
   class Project < Xcodeproj::Project
+
```
Comment (segiddins) [inline lib/cocoapods/project.rb]: no space here

**Comment by segiddins** [comment]
Seems fine, but it requires a rebase, and I'm not sure travis ran.

**Comment by alloy** [comment]
@segiddins Removed space and rebased, thanks for looking :+1:

**Comment by neonichu** [comment]
@samdmarshall :tada:

**Pull request merged**